CODEOWNERS

```diff
@@ -274,9 +274,9 @@ geode-core/**/org/apache/geode/internal/cache/control/**          @kirklund @Don
 # Testing utilities
 #-----------------------------------------------------------------
 #geode-concurrency-test/**
-geode-dunit/**                                                    @mhansonp @kirklund
```

Comment:
please add @demery-pivotal in place of yourself for geode-junit and geode-dunit (then tag him as a reviewer)

Reply:
done